var/spack/repos/builtin/packages/fftw/package.py

```diff
@@ -78,17 +79,21 @@ def install(self, spec, prefix):
 
         configure(*options)
         make()
+        make("check")
```

Comment:
@eschnett I agree with that sentiment. For the record, if `make check` fails, the build fails. I have noticed a lot of packages where the test fails, usually due to roundoff error or shoddy coding of tests that only work on one developer's computer. I usually try to add those tests but comment them out and mention that they fail and that I contacted the developers about it.

(1) If I recall, fftw took a very very long time to build. Comparatively, the tests ran fairly quickly. But please correct me if I'm wrong.

(2) Reliability is pretty much impossible to gauge. They passed for me. It's impossible to know if they pass for everyone.

I was hoping to wait until #1186 was merged before adding `self.run_tests` everywhere, since #1186 makes that obsolete. But if you guys want, I'm fine with using `self.run_tests` for this package.